src/westeros/renderer-backend.cpp

```diff
@@ -227,6 +236,8 @@ Backend::~Backend()
 void Backend::invalidate()
 {
     if (m_eventSource) {
+        if (m_eventSource->prepared && m_display)
+            wl_display_cancel_read(display);
```

Comment:
This probably should use m_display

Reply:
or m_eventSource->display

Reply:
Thanks Eugene

Of course, you're right

Fixed